base/14.16.0/Dockerfile

```diff
@@ -5,7 +5,7 @@
 # build it with command
 #   docker build -t cypress/base:14.16.0 .
 #
-FROM node:14.16.0-buster
+FROM node:14.16.0-buster-slim
```

Comment:
Wouldn't it be possible to base it on an alpine image instead of Debian?

Reply:
That would be a massive BC break, and a lot of added complexity to get chrome/firefox working there

Reply:
Ok. The complexity I get, but why a BC break? if the versions installed are the same at the end?

Reply:
Got it.